### ISSUE TITLE
gazebo_ros : starting gzserver and gzclient separately causes node name collision

### ISSUE THREAD

**Issue by wfvining** [ros-simulation/gazebo_ros_pkgs#765]
When attempting to the gazebo server separately from the gazebo client like so:

```
$ rosrun gazebo_ros gzserver
```

followed by

```
$ rosrun gazebo_ros gzclient
```

ROS gets a shutdown request for the `/gazebo` node because both gzserver and gzclient attempt to register a node with the same name (`/gazebo`).

**Comment by wfvining** [comment]
Sorry. I forgot to list versions:

This is with kinetic and gazebo version 7 on ubuntu 16.04

**Comment by kev-the-dev** [comment]
Yes it appears they both default to gazebo when you don't use the launch file.
In the mean time, you can run
```rosrun gazebo_ros gzclient __name:=gzclient```
